### DISCUSSION TITLE
How was "Strange Loop: Introduction To VisiData"?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1138] [General]
I was wondering if anyone who participated in this workshop wanted to share their impressions.  I'll assert that it was the biggest gathering of VisiData users.

**Comment by saulpw** [comment]
I'm not sure if any of the attendees are paying attention here on Github, but if they are, I'd love to hear their impressions as well.  I had a good time, for sure: it was really great getting a chance to expose some new people to the powers of VisiData, and to meet @jsvine in person.

**Reply by frosencrantz** [reply]
Glad to hear you and @jsvine had a great time.  Anything more you can say about the attendees? Like what they wanted to learn about Visidata, what kinds of professions, what kinds of data they wanted to explore.  It sounds like they were all new to Visidata.

**Reply by se16n** [reply]
Hi Saul @saulpw

Is there a chance recording can be viewed somewhere?

**Reply by saulpw** [reply]
Hi @se16n, unlike the talks at StrangeLoop, the workshops are not recorded.  You'd have to attend in person :)
